Source/WebKit/UIProcess/WebPageProxyInternals.h

```diff
@@ -294,7 +294,8 @@ struct WebPageProxy::Internals final : WebPopupMenuProxy::Client
 #endif
 
 #if ENABLE(WRITING_TOOLS)
-    HashMap<WTF::UUID, WebCore::TextIndicatorData> textIndicatorDataForChunk;
+    HashMap<WTF::UUID, WebCore::TextIndicatorData> textIndicatorDataForAnimationID;
+    HashMap<WTF::UUID, CompletionHandler<void()>> completionHandlerForAnimationID;
```

Comment:
Asked a question to the team about whether this should be `Function` rather than `CompletionHandler`.

Reply:
We are leaving as `CompletionHandler` since it's expected to be called in all cases except where the web process is killed.